components/fxa-client/src/lib.rs

```diff
@@ -84,14 +83,15 @@ pub(crate) struct StateV2 {
     #[serde(default)] // Same
     device_capabilities: HashSet<DeviceCapability>,
     session_token: Option<String>, // Hex-formatted string.
+    #[serde(default)] // Same
```

Comment:
micronit: maybe move this to before session_token so that it's more clear what `// Same` refers to.

Reply:
I wasn't sure if the order of fields might be relevant in some serializations, but also I guess it doesn't matter because we only use JSON anyway.